### PR TITLE
Add a note that Grafana is only for cluster, not user metrics

### PR DESCRIPTION
**Pull request by rh-max**

### CODE REVIEW

**Comment by rh-max** [comment]
@bmcelvee Could you PTAL and merge this + cherry-pick into `enterprise-4.3` and `enterprise-4.4`? Thanks!

**Comment by rh-max** [comment]
@bmcelvee This received a technical review, and no further (QE) review needed.

File: modules/monitoring-accessing-the-metrics-of-your-service.adoc [hunk 1]
```diff
@@ -7,6 +7,11 @@
 
 Once you have enabled monitoring your own services, deployed a service, and set up metrics collection for it, you can access the metrics of the service as a cluster administrator, as a developer, or as a user with view permissions for the project.
 
+[NOTE]
+====
+The Grafana instance shipped within OpenShift Monitoring is read-only and displays only infrastructure-related dashboards.
```
Comment (bmcelvee) [inline modules/monitoring-accessing-the-metrics-of-your-service.adoc:12]: s/OpenShift/{product-title}

**Comment by bmcelvee** [comment]
Just one change request, then I'll merge and cherry-pick

**Comment by rh-max** [comment]
@bmcelvee Done. Thanks!

**Pull request merged**